src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Cohort/DetailsViewModelMapperTests.cs

```diff
@@ -155,6 +164,27 @@ public void DraftApprenticeshipsAreMappedCorrectly()
             }
         }
 
+        [TestCase(0, "Approve 0 apprentices' details")]
+        [TestCase(1, "Approve apprentice details")]
+        [TestCase(2, "Approve 2 apprentices' details")]
+        public async Task PageTitleIsSetCorrectlyForTheNumberOfApprenticeships(int numberOfApprenticeships, string expectedPageTitle)
+        {
+            // Arrange
+            var draftApprenticeships = _autoFixture.CreateMany<DraftApprenticeshipDto>(numberOfApprenticeships).ToArray();
+            for (var i = 0; i < numberOfApprenticeships; i++)
+            {
+                SetCourseDetails(draftApprenticeships[i], $"Course1", "C1");
+            }
+            SetDraftApprenticeships(draftApprenticeships);
+
+            // Act_
+            var result = await _mapper.Map(_source);
+
+            // Assert
+
+            Assert.AreEqual(expectedPageTitle, result.PageTitle);
+        }
+
```

Comment:
I believe I refactored these unit tests in CV-9 to use the Fixture class design we agreed to follow. Would that style be a better way to write this tests as the mapper is getting quite complicated. I think these different approaches will cause a merge problem.

Reply:
Cheers - I'll fix these tests up when I merge them up.